amplify/function/BusinessLogic/AssignUsersToTeams/graphql/API.ts

```diff
@@ -2,64 +2,91 @@
 /* eslint-disable */
 //  This file was automatically generated and should not be edited.
 
-export type Team = {
-  __typename: "Team",
-  createdAt: string,
-  id: string,
-  members?: ModelUserConnection | null,
-  name?: string | null,
-  owner?: string | null,
-  updatedAt: string,
+export type GenericFunctionResponse = {
+  __typename: "GenericFunctionResponse";
+  body?: string | null;
+  headers?: string | null;
+  statusCode?: number | null;
```

Comment:
+1 Topan's comment

Reply:
These will be fixed once we set the fields to required in the model. They are not important here as the rest of the calls to the API are strongly typed using the internal TypeScript transformer provided by Amplify generateClient

Reply:
I think sometimes amplify doesn't update the model or is really wacky because I'm pretty sure I set the model on the FoodEvent as required on all the fields

Reply:
It was really hard to get this PR to be small since it was a culmination of 5 months of bug fixes and gradual improvements to a very large. For sure this would not happen again with chunking tickets into smaller branches

Reply:
to a very large ticket* I did not type that for some reason

Reply:
Also, let me know where the duplicated code is and I'll do my best to fix it :)

Reply:
@topanb11 With amplify v1 now a lot of lambda changes seem really huge but that's because each function has ~2k lines of autogenerated API code that has to be included with it. I believe @AllanKoder is going to delete some autogenerated forms which also contributes.